libraries/browser-tracker-core/src/tracker/out_queue.ts

```diff
@@ -420,7 +422,9 @@ export function OutQueueManager(
           if (beaconStatus === true) {
             onPostSuccess(numberToSend);
```

Comment:
Maybe in this case we should just call the onRequestSuccess as we won't get any more information from the beacon requests?